src/main/java/com/datbear/GuardiansOfTheRiftHelperPlugin.java

```diff
@@ -394,4 +368,17 @@ public void onMenuOptionClicked(MenuOptionClicked event)
 			}
 		}
 	}
+
+	@Subscribe
+	public void onOverheadTextChanged(OverheadTextChanged event)
```

Comment:
Make a PR with just this change in it and I will approve - I didn't know this was possible.